packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -417,6 +417,16 @@ export interface ICompressionRuntimeOptions {
 export interface IContainerRuntimeOptions {
 	readonly summaryOptions?: ISummaryRuntimeOptions;
 	readonly gcOptions?: IGCRuntimeOptions;
+
+	/**
+	 * When upgrading to 2.0, application authors need to defer the schema upgrade implicit in sending the GC op.
+	 * So provide a way to block the GC op (and all related features - GC Sweep/Tombstone) until ready for schema upgrade.
+	 *
+	 * If not true, the document schema will express support for GC op (updating if needed),
+	 * and any clients that don't support it will close on boot (e.g. 1.x clients or those still setting this to true)
+	 */
+	readonly blockGCOpForSchemaCompatibility?: boolean;
```

Comment:
So, default value means GC is on, right?
It's a bit hard to parse both comment and name of the property, maybe there is a better way to name it?

Reply:
Probably so :) open to suggestions. Was trying to be extra verbose / explicit while reasoning over it.